FirebaseDev.podspec

```diff
@@ -16,6 +16,7 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
   s.source           = { :git => 'https://github.com/firebase/firebase-ios-sdk.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/Firebase'
   s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.10'
```

Comment:
Unfortunately, that wouldn't work because of duplication between the FirebaseDev version of FirebaseCore and the published static library FirebaseCore that is a dependency of the missing static components.